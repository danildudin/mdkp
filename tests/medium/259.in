28 6 0
276 554 839 475 670 681 464 724 618 131 908 220 175 793 870 99 715 15 629 508 958 152 30 250 886 357 97 670 
61 295 329 651 685 394 746 417 943 527 730 349 391 922 261 919 835 582 900 98 899 208 630 919 708 412 210 646 
742 440 446 52 308 399 777 289 2 244 36 175 1 92 859 761 609 33 687 711 380 205 580 936 700 973 274 882 
755 62 234 986 635 134 36 591 261 174 143 203 158 984 706 807 160 161 138 14 575 510 872 960 158 967 925 950 
357 513 913 557 46 246 239 219 110 463 762 16 933 737 186 944 445 815 62 306 679 615 636 743 136 342 645 94 
773 624 402 746 178 357 734 487 295 62 315 363 402 373 923 813 280 990 953 411 751 402 171 619 884 84 894 367 
148 733 679 215 634 996 774 101 97 690 350 400 880 777 87 352 385 307 120 10 892 912 294 625 840 924 129 76 
9826 8506 9575 7239 5750 7564 
