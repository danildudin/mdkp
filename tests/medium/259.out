9689
13
2 4 5 7 8 10 13 14 16 20 24 25 27 
